ui/main_window.cpp

```diff
@@ -565,7 +565,7 @@ MainWindow::MainWindow(ApplicationController &controller) :
     // Retrieve the available metrics
     LoadAvailableMetrics();
 
-    m_trace_dig = new TraceDialog(m_controller, this);
+    m_trace_dig = new TraceDialog(m_controller, nullptr);
```

Comment:
Any reason it is unparented?

Reply:
looks like mac only allow parent window to have the close button